index.js

```diff
@@ -2,20 +2,19 @@ const Botkit = require('botkit')
 const config = require('./config')
 const fs = require('fs')
 const request = require('request')
+const WebClient = require('@slack/client').WebClient
 let projectList = null
 
+const token = process.env.SLACK_TOKEN || ''
+const web = new WebClient(token)
+
 const controller = Botkit.slackbot({
   debug: false,
   json_file_store: '/Users/johnokeeffe/dev/slackbot/users.json'
```

Comment:
This must use a relative path as it wouldn't work on someone else's machine. Try `./users.json`